postgres/src/crypto_trades.csv

```diff
@@ -0,0 +1,3157 @@
+"Date", "Timestamp", "Id", "Instrument", "Exchange", "Price", "Size"
```

Comment:
This feels like were starting to add demo data to the core repo, which I'm not sure that we want to do.  This seems more appropriate for the example repo.  The Dockerfile could even download from there.
@rcaudy thoughts?

Reply:
This is a set up that we do for Kafka testing as well. We do already store some test data in the repo (py/server/tests/data). I am not sure we have a clear rule about the generation/storage of test data.

Reply:
I'm ok having small, realistic, in-repo data for integration tests. That said, the test cases as currently written are very simplistic and don't seem to take advantage of the "realistic" data.